src/fedora/macros.tpl

```diff
@@ -0,0 +1,24 @@
+{% macro env_metadata() %}
```

Comment:
src/rhel is a symlink to src/centos, the macros.tpl file is used for both centos and rhel. This import https://github.com/mcyprian/s2i-python-container/commit/cabe677976b28eaca14f26d952fce561a7715460#diff-bd0e57c1ca2f20c05a1456d4512e2eafR2 wouldn't work otherwise.

Reply:
Ah, I completely overlooked that, thanks!